Corona-Warn-App/src/main/res/layout/include_certificate_overview_qr_card.xml

```diff
@@ -103,14 +102,11 @@
 
     <ImageButton
         android:id="@+id/covpass_info_button"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
-        android:layout_marginTop="11dp"
-        android:adjustViewBounds="true"
+        android:layout_width="48dp"
```

Comment:
Can you make it a style, like StatisticsCardInfoButton?

Reply:
Good idea - Added it.